test_scripts/API/RAI/052_ATF_P_steeringWheelLocation_LEFT_WARNINGS_RAI_steeringWheelLocation.lua

```diff
@@ -0,0 +1,263 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [RegisterAppInterface] SDL must send "steeringWheelLocation" param to each app in response
+-- [GetCapabilities] Conditions for SDL to store value of "steeringWheelLocation" param
+-- [MOBILE_API] [HMI_API] The 'steeringWheelLocation' enum
+-- [MOBILE_API] The 'steeringWheelLocation' param
+-- [HMI_API] The 'steeringWheelLocation' parameter
+-- [RegisterAppInterface] WARNINGS appHMIType(s) partially coincide or not coincide with current 
+-- non-empty data stored in PolicyTable
+--
+-- Description:
+-- In case any SDL-enabled app sends RegisterAppInterface_request to SDL
+-- and SDL has the value of "steeringWheelLocation" stored internally
+-- SDL must provide this value of "steeringWeelLocation" via RegisterAppInterface_response to mobile app
+--
+-- 1. Used preconditions
+-- Update value of "steeringWeelLocation" parameter from "HMI_capabilities.json" to RIGHT
+-- In InitHMI_OnReady HMI replies with parameter steeringWeelLocation = LEFT to UI.GetCapabilities
+--
+-- 2. Performed steps
+-- Register new applications with conditions for result WARNINGS
+--
+-- Expected result:
+-- SDL->mobile: RegisterAppInterface_response steeringWeelLocation is provided equal to LEFT
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForRAI = require('user_modules/shared_testcases/testCasesForRAI')
+local events = require("events")
+local mobile_session = require('mobile_session')
+
+--[[ Local functions ]]
+
+--[[@update_sdl_preloaded_pt_json - update preloaded_pt
+--! @ in case REAI to return response(WARNINGS, success:true)
+--! @parameters: NO
+--]]
+local function update_sdl_preloaded_pt_json()
+	local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+	local file = io.open(pathToFile, "r")
+	local json_data = file:read("*all") -- may be abbreviated to "*a";
+	file:close()
+				
+	local json = require("modules/json")
+	
+	local data = json.decode(json_data)
+	for k in pairs(data.policy_table.functional_groupings) do
+		if (data.policy_table.functional_groupings[k].rpcs == nil) then
+			data.policy_table.functional_groupings[k] = nil
+		end
+	end
+
+	data.policy_table.app_policies["0000001"] = {
+		keep_context = false,
+		steal_focus = false,
+		priority = "NONE",
+		default_hmi = "NONE",
+		groups = {"Base-4"}
+	}
+	data.policy_table.app_policies["0000001"].AppHMIType = {"NAVIGATION"}
```

Comment:
@istoimenova
I propose move AppHMIType parameter initialization to structure above:
```
	data.policy_table.app_policies["0000001"] = {
 		keep_context = false,
 		steal_focus = false,
 		priority = "NONE",
 		default_hmi = "NONE",
 		groups = {"Base-4"},
		AppHMIType = {"NAVIGATION"}
 	}
```